### PR TITLE
Removed temp fixes which were fixed in latest PatternFly release 

### LINKED ISSUE #1261: Remove ::placeholder vendor prefixes once updated in PatternFly
These rules included in https://github.com/openshift/origin-web-console/pull/1260 are added temporarily while awaiting an update to patternfly to fix https://github.com/patternfly/patternfly/issues/597

### LINKED ISSUE #1306: Remove bootstrap-switch dependency (or resolve jQuery conflict)
Follow on to #1302. See also issue #1301

bootstrap-switch 3.3.4 requires jQuery ~1.9.0, which is incompatible with the jQuery version from Patternfly. Look at removing the dependency since we're not using it directly.

### PR DESCRIPTION
**Pull request by dtaylor113**
Fixes #1306 Remove bootstrap-switch dependency

Fixes #1261 Remove ::placeholder vendor prefixes once updated in PatternFly

- Removes alignment of wizard step indicator number which was fixed in base PatternFly

### CODE REVIEW

File: dist.java/java/404.html [hunk 1]
```diff
@@ -38,7 +38,7 @@
       <div ng-include src="viewPartial"></div>
     </div>
 
-    <script src="libs-3239db0851.js"></script>
+    <script src="libs-cee83b5806.js"></script>
```
Comment (spadgett) [inline dist.java/java/404.html]: This diff shouldn't be here. I would clean and reinstall assets

Comment (dtaylor113) [inline dist.java/java/404.html]: Fixed, thanks!

**Comment by spadgett** [comment]
@dtaylor113 Looks like you need to add some bower resolutions (see the travis log)

**Comment by dtaylor113** [comment]
Wondering why, I didn't change jQuery?!   I'll add a resolution to bower.json.

**Comment by spadgett** [comment]
[test]

**Comment by spadgett** [comment]
[merge]

**Pull request merged**